nats-base-client/types.ts

```diff
@@ -136,18 +137,28 @@ export interface Base {
 }
 
 export interface ServerInfo {
+  server_id: string;
+  server_name: string;
+  version: string;
+  proto: number;
+  git_commit?: string;
+  go: string;
+  host: string;
+  port: number;
+  headers?: boolean;
+  auth_required?: boolean;
   tls_required?: boolean;
   tls_verify?: boolean;
-  connect_urls?: string[];
+  tls_available?: boolean;
   max_payload: number;
-  client_id: number;
```

Comment:
I already commented in the past that those constant re-ordering are making review difficult because I really have to scan to see what is added/removed vs just moved around.

Reply:
yes well - I needed to map them to the server.

Reply:
What I mean is that some move around and it is difficult to see which ones are added or removed, or simply moved. Why move the ones that did not change?

Reply:
They now are ordered in teh same way as the server. - I prefer alpha, but server is not sorted.

Reply:
You pick the order you want, they don't have to align with what is in the server's Info structure, but then please stick with it.